crates/vsock/src/vhu_vsock.rs

```diff
@@ -38,6 +38,9 @@ pub(crate) const BACKEND_EVENT: u16 = 3;
 /// TODO: Make this value configurable
 pub(crate) const CONN_TX_BUF_SIZE: u32 = 64 * 1024;
 
+/// At what point should we send a credit update.
+pub(crate) const CREDIT_UPDATE_THRESHOLD: u32 = CONN_TX_BUF_SIZE / 4;
```

Comment:
Can you explain why use `CONN_TX_BUF_SIZE / 4`?

Maybe we can use the same value we used in Linux, but if you prefer to use that value, please rebase on top of #324

Reply:
I was gonna add the link to it, in the Linux codebase the value is 1/4th of thr buffer size that's why I chose it

Reply:
Sorry, I can't see it in the Linux codebase, can you put a link?

Reply:
I think it is just a case that it is 1/4 of the default value of buf_alloc, but if buf_alloc is changed the threashold remains fixed. I think it makes sense to use the same value, or do some experiments to show that choosing 1/4 is okay.

Reply:
Yes the value is 1/4 of the max buf_alloc. The reason i kept it variable and actually dependent upon buf_alloc is because since in our case the allocated buffer could be of any size, assume it's very small and we have the threshold as quite high we would be again sending the updates on every read sort off. If we had a range for that then we could still try to make it a fixed value. I tried with different values 1/4 seems fine and produced good results. I have the numbers and was gonna post that as part of the benchmarks

Reply:
Okay, it makes sense to me ;-)